tsfc/fem.py

```diff
@@ -233,6 +234,11 @@ def flat_index(ordered_deriv):
         return result[0]
 
 
+def zero_tracking_literal(table, ufl_element):
+    track_zeros = isinstance(ufl_element, (ufl.EnrichedElement, ufl.MixedElement))
```

Comment:
Maybe I missed something.  I thought Fabio only ever stripped away initial and final columns.  I didn't think there was any risk of looping over, e.g. columns 1-4, 6, 8-10, 12-16 [if cols 5, 7 and 11 'happen' to be 0].